ads/nativeroll.js

```diff
@@ -39,6 +39,9 @@ function initPlayer(data) {
     container: '#c',
     desiredOffset: 50,
     gid: data.gid,
+    onError: () => {
+      window.context.noContentAvailable();
```

Comment:
please pass down the `window` from caller (`global`)

Reply:
done

Reply:
Or i need something like "global.window.contex..."?

Reply:
```
function initPlayer(global, data) {
  ...
  global.context.noContentAvailable();
}
```